examples/Session05/maillroom_test.py

```diff
@@ -174,6 +174,16 @@ def input_sc(textfile):
     while bool:
         print(welcome)
         selection = input('Please type a number between 1 & 4:\n')
+<<<<<<< HEAD
```

Comment:
you should be able to put this back the way it was with:

```
git reset --hard  d9a39776abedaf2d  examples/Session05/maillroom_test.py
```
That weird string of characters is the "hash" for a recent "commit" --or version of the repository.

So you are saying -- reser this file to the old version. the --hard means "even if I've changed it locally"

Reply:
hmmmm, I'm getting a 'fatal: Cannot do hard reset with paths.'

Reply:
hmm, hard to test, as I don't have the same setup -- but I"ll try to figure out the right way...

Reply:
OK -- I didn't read far enoughin that SO post -- turns out that "reset" does NOT work for individual files -- as you discovered. The correct command is:

`git checkout 8e5908a37d7d examples/Session05/maillroom_test.py`

which I should have known. `git checkout filename` will put a file back to the state of the previous commit. In this case, you've committed that change, so you need to go further back.